src/components/AdminSettings.vue

```diff
@@ -12,16 +12,48 @@
 				</label>
 				<input id="collaboard-instance"
 					v-model="state.admin_instance_url"
+					:class="{ 'greyed-out-text': !usingCustomInstanceUrl, 'invalid-url': !isValidUrl(state.admin_instance_url) }"
```

Comment:
Would be cleaner with a computed prop like `isInstanceUrlValid` which calls `isValidUrl`. It's not clear that Vue is reactive when written like it is now.

Reply:
Ah, didn't quite think that one through! But seems likely it won't/might not be responsive.